types/oauth2orize/.eslintrc.json

```diff
@@ -1,5 +1,6 @@
 {
     "rules": {
-        "@definitelytyped/no-any-union": "off"
+        "@definitelytyped/no-any-union": "off",
+        "@definitelytyped/no-unnecessary-generics": "off"
```

Comment:
```suggestion
```

remove this and fix error or disable it on a line that needs to be.

Reply:
Changed by ab3d655213bead94dfb8f269182ed6366887979d
I disabled this rule as it errored at lots of places. It does not honor inheritance of types as a reason to keep it in.